.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,111 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: test package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    env:
+     VERMOUTH_TEST_DSSP: mkdssp
+     SKIP_GENERATE_AUTHORS: 1
+     SKIP_WRITE_GIT_CHANGELOG: 1
+    
+    strategy:
+      matrix:
+       py_version: ["3.6", "3.7", "3.8", "3.9", "3.6-dev", "3.7-dev", "3.8-dev", "3.9-dev"] 
+       include:
+         - py_version: "3.6"
+           WITH_CODECOV: true
+           WITH_SCIPY: true
+         - py_version: "3.7"
+           WITH_CODECOV: true
+           WITH_SCIPY: true
+         - py_version: "3.8"
+           WITH_CODECOV: true
+           WITH_SCIPY: true
+         - py_version: "3.9"
+           WITH_CODECOV: true
+           WITH_SCIPY: true
+         - py_version: "3.6-dev"
+           WITH_CODECOV: true
+           WITH_SCIPY: true  
+         - py_version: "3.7-dev"
+           WITH_CODECOV: true
+           WITH_SCIPY: true   
+         - py_version: "3.8-dev"
+           WITH_CODECOV: false
+           WITH_SCIPY: false
+         - py_version: "3.9-dev"
+           WITH_CODECOV: false
+           WITH_SCIPY: false
```

Comment:
About the build matrix:
- Coverage used to be wrong (too low) for py>38, which is why we disabled coverage reporting for it. Similar for py39. That seems fixed now, so we could remove the variable.
- We need 1 case where we test without scipy. I suggest adding a py39 (not dev) without scipy, and testing 38dev and 39dev with scipy. 

Also, this matrix is actually quite large. How do you feel about removing py36-dev and py37-dev?

Reply:
fine by me also these dev versions are not really a thing anymore. Or at least they should be reasonably in line with 3.7 and 3.6. So I axed them and took care of the scipy thing.